pages/pipelines/dependencies.md.erb

```diff
@@ -104,7 +104,7 @@ Steps that are in a `concurrency_group` run in the order they are created in and
 
 ## Allowing dependency failures
 
-You can add the `allow_dependency_failure` attribute to any step that has dependencies. The step will then run when the depended-on jobs complete, even if those jobs have failed. However, even if you continue to run the next step, if there are any failures at all, the build will still end up failing.
+You can add the `allow_dependency_failure` attribute to any step that has dependencies. The step will then run when the depended-on jobs finish, even if those jobs have failed or did not run. However, even if you continue to run the next step, if there are any failures at all, the build will still end up failing.
```

Comment:
[`finished` is a job state](https://buildkite.com/user/graphql/documentation/type/JobStates), so I'd hesitate to use that word here to more generally mean "reached one of the terminal job states"

Reply:
:100: thanks @matthewd I'll fix up